fetch/api/cors/cors-preflight-star.any.js

```diff
@@ -35,8 +35,11 @@ function preflightTest(succeeds, withCredentials, allowMethod, allowHeader, useM
 preflightTest(true, false, "get", "x-test", "GET", ["X-Test", "1"])
 preflightTest(true, false, "*", "x-test", "SUPER", ["X-Test", "1"])
 preflightTest(true, false, "*", "*", "OK", ["X-Test", "1"])
+preflightTest(true,  true, "OK", "Content-Type", "GET", ["Content-Type", "uni/corn"])
```

Comment:
This test doesn't appear to test *-handling so it should probably move to cors-preflight.any.js or some such.